l2geth/rollup/sync_service.go

```diff
@@ -157,7 +165,8 @@ func NewSyncService(ctx context.Context, cfg Config, txpool *core.TxPool, bc *co
 	// a remote server that indexes the layer one contracts. Place this
 	// code behind this if statement so that this can run without the
 	// requirement of the remote server being up.
-	if service.enable {
+	// If we're syncing from the Queue, then we can skip all this and rely on L2 published transactions
+	if service.enable && service.backend != BackendQueue {
```

Comment:
Yay or nay? Side effect of this is that `eth_syncing` always returns false when we're syncing from the Queue. IsSyncing is only used to prevent transactions from being submitted to the node. But only verifiers will use the BackendQueue, so I don't think this would be an issue.

Reply:
I think this is OK - I don't expect any replica operators to use `BackendQueue`.